modules/openapi/tests/src/StructureSpec.scala

```diff
@@ -384,4 +384,29 @@ final class StructureSpec extends munit.FunSuite {
     TestUtils.runConversionTest(openapiString, expectedString)
   }
 
+  test("structures - property name sanitization") {
+    val openapiString = """|openapi: '3.0.'
+                     |info:
+                     |  title: test
+                     |  version: '1.0'
+                     |paths: {}
+                     |components:
+                     |  schemas:
+                     |    Object:
+                     |      type: object
+                     |      properties:
+                     |        version3.1:
+                     |          type: string
+                     |""".stripMargin
+
+    val expectedString = """|namespace foo
+                      |
+                      |structure Object {
+                      | version31: String
```

Comment:
should this have a `@jsonName("version3.1")`?

Reply:
Ah yeah, I totally overlooked that. Thanks will add

Reply:
also worth checking if this affects unions / other places that `@jsonName` is viable in :)

Reply:
Added for structures in https://github.com/disneystreaming/smithy-translate/pull/48/commits/49fb2075405e102312705bce73dd8ee8c6878bbd will look at unions now

Reply:
Okay and done for unions in https://github.com/disneystreaming/smithy-translate/pull/48/commits/c171da9b107b8050b600770575946fb1c7d08951

Reply:
neato!